### PR TITLE
Essentials deploy script

### PR DESCRIPTION
**Pull request by borditamas**

### CODE REVIEW

File: core-common/essentials-deploy/deploy.bat [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@echo OFF
+cls
+
+:start
+set /p VERSION=Type the version:
+if [%VERSION%]==[] goto start
+
+if not exist .\work md "work"
+cd .\work
+del /Q *.*
```
Comment (rbocsi) [inline core-common/essentials-deploy/deploy.bat]: it is very dangerous to delete `*.*`. 

Maybe you should use 

```
rmdir /Q /S work
md work
```
instead.

File: core-common/essentials-deploy/deploy.bat [hunk 2]
```diff
@@ -0,0 +1,56 @@
+@echo OFF
```
Comment (rbocsi) [inline core-common/essentials-deploy/deploy.bat:2]: A Linux-version would be good too.

**Review by rbocsi — CHANGES_REQUESTED** [review]
see inline comments

File: core-common/essentials-deploy/deploy.bat (hunk 1 above)
Comment (borditamas) [reply core-common/essentials-deploy/deploy.bat]: done

File: core-common/essentials-deploy/deploy.bat (hunk 2 above)
Comment (borditamas) [reply core-common/essentials-deploy/deploy.bat:2]: I've put a TODO remark into the batch script

**Review by rbocsi — APPROVED** [review]

**Review by MaGaMeGa — APPROVED** [review]

**Pull request merged**